### PR TITLE
Toggle menu

### PR DESCRIPTION
**Pull request by rpire**
## Changes made:

- Implemented an on-click toggle menu for the mobile version.
- Added script.js file.
- Added Linter for JavaScript.
- Added internal links to the navigation bar.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -599,10 +636,37 @@ footer ul {
...
+    display: none;
+  }
+
+  .deco-b {
+    display: none;
+  }
+
+  .mos-0 {
+    display: none;
+  }
```
Comment (Dipeshtwis) [inline style.css:668]: - [ ] I would recommend to follow the DRY (Do not repeat yourself) principle to follow the CSS best practices. Here you have gave the same property to three different classes so you can choose something like this.

```
.class-a,
.class-b,
.class-c {
   ...... // any property
 }
```

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @rpire, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: On desktop size the hamburger menu button is not visible and on mobile the hamburger menu button is visible.
:+1: the hamburger button and the close (X) button working fine.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rpire** [comment]
## Greetings @Dipeshtwis 

Thank you for reviewing my work, I will make sure to implement the requested changes and ask for another review. Also, I will consider this advice for my future projects.  🤓

Cheers and Happy coding! 🚀🚀

File: style.css (hunk 1 above)
Comment (rpire) [reply style.css]: Request satisfied in last commit.

Comment (rloterh) [reply style.css]: This issue was well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Rubén, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾👍.
Congratulations! 🎉🥂


Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rpire** [comment]
## Thank you!

Hello @rloterh 

Thank you for reviewing my work. I will proceed to merge. 🙌

Once again thank you, Cheers and Happy coding! 🚀🚀

**Pull request merged**